packages/main/src/RPA/Robocorp/Process.py

```diff
@@ -163,59 +164,63 @@ def __init__(
         self.http = HTTP()
 
     @keyword(tags=["set"])
-    def set_workspace_id(self, workspace_id: str = None):
+    def set_workspace_id(self, workspace_id: str = None) -> None:
```

Comment:
```suggestion
    def set_workspace_id(self, workspace_id: Optional[str] = None) -> None:
```

Reply:
And for all the other similar cases :). (don't forget to `from typing import Optional` as well)